src/database-sqlite.ts

```diff
@@ -34,22 +34,22 @@ export async function initStorage (options: Record<string, unknown> = {}): Promi
   if (!options.internal) {
     console.log('Connected to the %s SQLite database.', filepath)
   }
-  iterKeysStatement = db.prepare<[string]>('SELECT key FROM Data')
-  readStatement = db.prepare<[string]>('SELECT value FROM Data WHERE key = ?')
+  iterKeysStatement = db.prepare('SELECT key FROM Data')
+  readStatement = db.prepare('SELECT value FROM Data WHERE key = ?')
   // Use "upsert" syntax to store an entry only if the key is not already in the DB.
   // https://www.sqlite.org/lang_upsert.html
   writeOnceStatement = db.prepare('INSERT INTO Data(key, value) VALUES(?, ?) ON CONFLICT (key) DO NOTHING')
   writeStatement = db.prepare('REPLACE INTO Data(key, value) VALUES(?, ?)')
 }
 
 export function count (): number {
-  return db.query<[number]>('SELECT COUNT(*) FROM Data')[0][0]
+  return db.prepare('SELECT COUNT(*) FROM Data').all()[0][0]
 }
 
 // deno-lint-ignore require-await
 export async function readData (key: string): Promise<Uint8Array | string | void> {
   // For some reason `[null]` is returned when the value is an empty Uint8Array.
-  const maybeRow = readStatement.first([key])
+  const maybeRow = readStatement.all([key])[0]
```

Comment:
Statement.first() is removed; used .all() and accessed the first element manually.

Reply:
Is this guaranteed to be defined?

Reply:
Right, it's not guaranteed to be defined. If the query doesn't find a row matching the key, readStatement.all([key]) will return an empty array.

We should do `const maybeRow = readStatement.all([key])[0] ?? undefined`

Reply:
Hmm, if it returns an empty array, is it necessary to do that?

```
> [][0] ?? undefined
undefined
> [][0]
undefined
> 
```

Reply:
https://github.com/okTurtles/chel/pull/46/commits/f6dde63cb1ae5875b056c3ddf067edd8049822da

Reply:
Sorry if it wasn't clear. https://github.com/okTurtles/chel/commit/f6dde63cb1ae5875b056c3ddf067edd8049822da should be reverted because what it replaced has the same effect. My question was about whether `readStatement.all([key])[0]` would always succeed (e.g., maybe `readStatement.all([key])` was `undefined` or `null`).

Reply:
Should we keep as is?
`const maybeRow = readStatement.all([key])[0]`

Reply:
Reverted https://github.com/okTurtles/chel/pull/46/commits/3cedbd2bcd313d6852344306f7829d61d592504e